homework-12/main.js

```diff
@@ -0,0 +1,74 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let firstNumber = '';
+    let secondNumber = '';
+    let signOperation = '';
+    let finish = false;
+    const digit = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.'];
+    const action = ['-', '+', 'x', '/'];
+    const out = document.querySelector('.calc-screen p');
+
+    document.querySelector('.ac').addEventListener('click', () => {
```

Comment:
Nice to have a variable for an element and then assign a listener for it, just to keep it easy to read